index.bs

```diff
@@ -2205,7 +2206,7 @@ Integrations {#integrations}
 Feature Policy {#feature-policy}
 --------------
 
-This specification defines a [=policy-controlled feature=] that controls whether the {{Navigator/xr}} attribute is exposed on the {{Navigator}} object.
+This specification defines a [=policy-controlled feature=] that controls whether the any {{XRSession}}s may be returned by {{XR/requestSession()}} and whether {{XR/supportsSession()}} indicates that any session mode is supported.
```

Comment:
We should also consider the behavior of device change events

Reply:
Great catch! I've updated the PR to also indicate that `devicechange` events shouldn't fire when the feature policy is not set.